arc/database/client.py

```
@@ -33,6 +33,23 @@
 T = TypeVar('T')
 
 
+def convert_row_to_dict(row) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)